spring-integration-core/src/main/java/org/springframework/integration/config/IntegrationManagementConfigurer.java

```diff
@@ -90,7 +90,7 @@
 
 	private Boolean defaultStatsEnabled = false;
 
-	private MetricsFactory metricsFactory;
+	private org.springframework.integration.support.management.MetricsFactory metricsFactory;
```

Comment:
The `setMetricsFactory`, `setMetricsFactoryBeanName` and I guess `setEnabledStatsPatterns` and `setDefaultStatsEnabled` must be deprecated in this class as well.

The same about `stats` and factory I guess is applied for the `@EnableIntegrationManagement`